repo/packages/M/marathon/0/config.json

```diff
@@ -0,0 +1,147 @@
+      "maximum": 65535
+    },
+    "marathon/mesos-role": {
+      "description": "Mesos role for this framework.",
+      "type": "string"
+    },
+    "marathon/reconciliation-initial-delay": {
+      "description": "The delay, in milliseconds, before Marathon begins to periodically perform task reconciliation operations.",
+      "type": "integer",
+      "minimum": 0
+    },
+    "marathon/reconciliation-interval": {
+      "description": "The period, in milliseconds, between task reconciliation operations.",
+      "type": "integer",
+      "minimum": 0
+    },
+    "marathon/event-subscriber": {
+      "description": "Event subscriber module to enable. Currently the only valid value is \"http_callback\".",
+      "type": "string"
+    },
+    "marathon/http-endpoints": {
+      "description": "Pre-configured http callback URLs. Valid only in conjunction with --event_subscriber http_callback. Additional callback URLs may also be set dynamically via the REST API.",
+      "type": "string"
+    },
+    "marathon/zk": {
+      "description": "ZooKeeper URL for storing state. Format: zk://host1:port1,host2:port2,.../path",
+      "type": "string"
+    },
+    "marathon/zk-max-versions": {
+      "description": "Limit the number of versions stored for one entity.",
+      "type": "integer",
+      "minimum": 0
+    },
+    "marathon/zk-timeout": {
+      "description": "Timeout for ZooKeeper in milliseconds.",
+      "type": "integer",
+      "minimum": 0
+    },
+    "marathon/mesos-authentication-principal": {
+      "description": "The Mesos principal used for authentication.",
+      "type": "string"
+    },
+    "marathon/mesos-authentication-secret-file": {
+      "description": "The path to the Mesos secret file containing the authentication secret.",
+      "type": "string"
+    },
+    "marathon/marathon-store-timeout": {
+      "description": "Maximum time in milliseconds, to wait for persistent storage operations to complete.",
+      "type": "integer",
+      "minimum": 0
+    },
+    "marathon/http-credentials": {
+      "description": "Credentials for accessing the HTTP service in the format of username:password. The username may not contain a colon (:).",
+      "type": "string"
+    },
+    "marathon/http-port": {
+      "description": "The port on which to listen for HTTP requests.",
+      "type": "integer",
+      "minumum": 0,
+      "default": 8080
+    },
+    "marathon/https-port": {
+      "description": "The port on which to listen for HTTPS requests.",
+      "type": "integer",
+      "minumum": 0
+    },
+    "marathon/ssl-keystore-password": {
+      "description": "Password for the keystore supplied with the marathon/ssl-keystore-path option.",
+      "type": "string"
+    },
+    "marathon/ssl-keystore-path": {
+      "description": "Path to the SSL keystore. SSL will be enabled if this option is supplied.",
+      "type": "string"
+    }
+  },
+  "required": [
+    "mesos/master",
+    "marathon/zk"
```

Comment:
I see you have {#...}...{/...} in the marathon app resource. Based on that, it looks like we require: `marathon/cpus`, `marathon/mem`, `marathon/instances`, `marathon/http-port`. It doesn't look like `meso/master` and `marathon/zk` are required.